package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "earthbnb-frontend",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "eslint": "npx eslint \"**/*.{js,jsx}\"",
+    "stylelint": "npx stylelint \"**/*.{css,scss}\""
```

Comment:
- It looks like there isn't a way to run tests, something like `npm run test`, I kindly invite you to add the commando to run tests properly on the app. 🐱